engine/emoji-picker

```diff
@@ -16,8 +16,8 @@
 #  You should have received a copy of the GNU General Public License
```

Comment:
Oh, I made a mistake, this file is generated and shouldn’t have been in git.
I removed it now and added it to .gitignore.